### PR TITLE
Created a Form and Dashboard for Feature Toggle

### PR DESCRIPTION
**Pull request by ASHWINIAJIT**
I have created a form and a dashboard for Feature Toggle!

### CODE REVIEW

**Comment by ASHWINIAJIT** [comment]
@Cartmanishere @ankushdharkar 

As we discussed, I have refactored the code using template tag.

Please review the PR!

**Comment by ASHWINIAJIT** [comment]
@Cartmanishere @ankushdharkar 

As discussed, I have removed the owner field and refactored the code.

Please check the PR!

**Comment by ASHWINIAJIT** [comment]
@Cartmanishere @ankushdharkar 

I have added the event handler for feature toggle.

Please check the PR and let me know if anything needs to be changed!

File: featureFlag/dashboard/index.js [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+        config: { enabled: false }
+      }
+    } else {
+      dataToBeSent = {
+        config: { enabled: true }
+      }
+    }
+
+    try {
+      const response = await fetch(`${BASE_URL}/featureFlags/` + featureId, {
```
Comment (Cartmanishere) [inline featureFlag/dashboard/index.js:134]: You can include `featureId` in template literal. 

`${BASE_URL}/featureFlags/${featureId}`

File: featureFlag/dashboard/index.js [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+        method: "PATCH",
+        credentials: "include",
+        body: JSON.stringify(dataToBeSent),
+        headers: {
+          "Content-type": "application/json",
+        },
+      });
+      const result = await response.json();
+      alert(result.message);
+      changeTheFeatureToggle(featureId);
```
Comment (Cartmanishere) [inline featureFlag/dashboard/index.js:144]: Check the response code before changing the feature toggle.

What if there was an error on backend and the request returned non-successful status code.

File: featureFlag/dashboard/index.html [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+</head>
+
+<body>
+  <div id="container">    
+      <p class="heading">Features:</p>     
+  </div>
+</body>
+
+<script src='./index.js'></script>
+</html>
```
[the remark this answers is not in the document]
Comment (ASHWINIAJIT) [reply featureFlag/dashboard/index.html]: Added the new line at the end .

File: featureFlag/dashboard/index.js (hunk 1 above)
Comment (ASHWINIAJIT) [reply featureFlag/dashboard/index.js]: Changed the code as suggested.

File: featureFlag/dashboard/index.js (hunk 2 above)
Comment (ASHWINIAJIT) [reply featureFlag/dashboard/index.js]: Verified the status code before changing the feature toggle.

**Comment by ASHWINIAJIT** [comment]
@Cartmanishere @ankushdharkar @Ankita2002-Fr 

I have raised a PR with all suggested changes.

Please check!

**Review by Cartmanishere — APPROVED** [review]

**Pull request merged**